src/components/Storyblok/SbTriangle.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { storyblokEditable } from 'gatsby-source-storyblok';
+import { Triangle } from '../Shapes';
+
+type SbTriangleProps = {
+  blok: {
+    _uid: string;
+  };
+};
+
+export const SbTriangle = ({
+  blok: {
+    _uid,
+  },
+  blok,
+}: SbTriangleProps) => (
+  <Triangle {...storyblokEditable(blok)} key={_uid} />
+);
```

Comment:
Add this Triangle component on Storyblok since we're using it in a lot of places. This would allow for the content editor to pop it in to the WYSIWYG component for example